right_column.md

```diff
@@ -75,29 +75,39 @@ Python.
   [text instructions](https://coderefinery.github.io/installation/git/#configuring-git),
   [video](https://www.youtube.com/watch?v=WdDTp8NeHBs&t=258s).
 
+
+### Host
+- Naoe Tatara
```

Comment:
```suggestion
### Organizers
- Naoe Tatara
- Richard Darst
```
see below for reasoning

Reply:
I have no objection that you will be an organizer, but am I qualified to be listed there as well?? I follow your decision here :)

Reply:
I think so!  At the very minimum, who should be asked in case of issues during the workshop?  One of us, so I think this is a useful distinction no matter what.  In addition to other things too.